config/locales/en.yml

```diff
@@ -238,6 +242,17 @@ en:
           private_nursery: "Private nursery"
           another_early_years_setting: "Another early years setting"
 
+        course_id_options:
+          2: "Leading behaviour and culture"
+          3: "Leading literacy"
+          4: "Leading teaching"
+          5: "Leading teacher development"
+          6: "Senior leadership"
+          7: "Headship"
+          8: "Executive leadership"
+          9: "Early years leadership"
+          10: "Early headship coaching offer"
```

Comment:
You're right that using IDs isn't ideal, we should return to this and implement your course.key system so that we can stop using primary keys anywhere. It'd also let us stop referencing ECF UUIDs within the NPQ app at all if we were just passing keys over to ECF and it used that to look up records. 

It'd allow us to cut that hard link between database records!